patches/gtsam.patch

```diff
@@ -1,13 +1,92 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8c4229e..5ab20aa 100644
 --- ./CMakeLists.txt
 +++ ./CMakeLists.txt
-@@ -207,7 +207,7 @@ endif()
- ### use our patched version of Eigen
- ### See:  http://eigen.tuxfamily.org/bz/show_bug.cgi?id=704 (Householder QR MKL selection)
- ###       http://eigen.tuxfamily.org/bz/show_bug.cgi?id=705 (Fix MKL LLT return code)
--option(GTSAM_USE_SYSTEM_EIGEN "Find and use system-installed Eigen. If 'off', use the one bundled with GTSAM" OFF)
-+option(GTSAM_USE_SYSTEM_EIGEN "Find and use system-installed Eigen. If 'off', use the one bundled with GTSAM" ON)
```

Comment:
(switch default from OFF to ON) does not appear at final status for patches/gtsam.patch. This part is removed
\-  -
\- +

If we are talking about patches/gtsam.patch. The final file status after locally checking this PR:

`$ git fetch origin pull/112/head:pull_112` 
`$ git chekcout pull_112` 
`$ cat patches/gtsam.patch`
`diff --git a/CMakeLists.txt b/CMakeLists.txt`
`index 8c4229e..5ab20aa 100644`
` --- ./CMakeLists.txt`
` +++ ./CMakeLists.txt`
`@@ -225,6 +225,8 @@ option(GTSAM_USE_SYSTEM_EIGEN "Find and use system-installed Eigen. If 'off', us`
` if(GTSAM_USE_SYSTEM_EIGEN)`
` 	find_package(Eigen3 REQUIRED)`
` 	include_directories(AFTER "${EIGEN3_INCLUDE_DIR}")`
`+        find_package(LAPACK REQUIRED)`
`+`
 
` 	# Use generic Eigen include paths e.g. <Eigen/Core>`
`     set(GTSAM_EIGEN_INCLUDE_PREFIX "${EIGEN3_INCLUDE_DIR}")`
`@@ -335,6 +337,10 @@ if(GTSAM_ENABLE_CONSISTENCY_CHECKS)`
`   add_definitions(-DGTSAM_EXTRA_CONSISTENCY_CHECKS)`
` endif()`
 
`+if(NOT WIN32)`
`+    option(GTSAM_BUILD_PKGCONFIG "Build pkg-config .pc file for GTSAM" ON)`
`+endif(NOT WIN32)`
`+`
`###############################################################################`
` # Add components`

`diff --git a/gtsam/CMakeLists.txt b/gtsam/CMakeLists.txt`